4food/src/reducers/user.js

```diff
@@ -0,0 +1,15 @@
+const initialState = {
+}
+
+
+const user = (state = initialState, action) => {
+    switch(action.type){
+        case "LOGIN_USER":
+            return {...state, allLogin: action.payload.login}
```

Comment:
Acredito que um nome mais significativo poderia ser:

```suggestion
            return {...state, loginInformation: action.payload.login}
```